libs/coin-modules/coin-kaspa/src/bridge/getTransactionStatus.ts

```diff
@@ -83,10 +88,17 @@ const getTransactionStatus = async (
       if (transaction.amount.lt(20000000)) {
         errors.dustLimit = new DustLimit("");
       }
+      if (utxos.length > MAX_UTXOS_PER_TX) {
+        warnings.amount = new ReducedAmountUtxoWarning();
+      }
     }
 
     if (transaction.amount.gt(maxSpendableAmount) || maxSpendableAmount.eq(0)) {
-      errors.amount = new NotEnoughBalance();
+      if (utxos.length > MAX_UTXOS_PER_TX && account.balance.gt(transaction.amount)) {
```

Comment:
I don't understand how can `account.balance.gt(transaction.amount)` be true if in this scope `transaction.amount.gt(maxSpendableAmount) || maxSpendableAmount.eq(0)`.

Isn't this always false?